.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: Create Checksum
+          command: sh .circleci/checksum.sh /tmp/checksums.txt
+      - restore_cache:
+          keys:
+            - npm-cache-{{ checksum "/tmp/checksums.txt" }}
+      - run:
+          name: Install Dependencies
+          command: yarn install
```

Comment:
Should this be `lerna bootstrap` instead? How are individual packages installed and linked together with `yarn install`?

Reply:
Another option is to add  `"postinstall": "yarn run bootstrap"` script.

Reply:
Looks like we already have this option: https://github.com/open-telemetry/opentelemetry-js/blob/master/package.json#L9